Engine_Revit_UI/Convert/Geometry/FromRevit/Curve.cs

```diff
@@ -191,14 +191,13 @@ public static oM.Geometry.ICurve IFromRevit(this Curve curve)
 
             if (result == null)
             {
+                BH.Engine.Reflection.Compute.RecordWarning("Curve types without conversion support have been tesellated and converted into Polylines.");
```

Comment:
Spelling, `tesellated` should be `tessellated`

Reply:
I had a feeling that it will be wrong 🤣 Fixed.